bedrock/firefox/templates/firefox/concerts.html

```diff
@@ -2,48 +2,189 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "firefox/base-pebbles.html" %}
+{% extends "firefox/base-protocol.html" %}
 
-{% block page_title %}Give email, watch concert{% endblock %}
+{% block page_title_prefix %}Firefox Concert Series - {% endblock %}
+{% block page_title %}Watch free live concerts in Firefox{% endblock %}
+{% block page_desc %}Here’s to you. Here’s to live music. Here’s to watching free, from all kinds of artists, right from Firefox. Thanks for being part of our community. Firefox fights for you.{% endblock %}
 
 {% block page_css %}
-  {{ css_bundle('firefox_concerts') }}
+  {{ css_bundle('firefox_concert_series') }}
 {% endblock %}
 
+{% block body_class %}firefox-concerts{% endblock %}
+
 {% block content %}
 <main role="main">
-  <div class="content">
-    <div class="conditional-content verified-content">
-      <p>You have been verified!</p>
+  <div class="concert-content">
+    <header class="mzp-c-hero mzp-t-firefox mzp-t-product-firefox mzp-t-dark">
```

Comment:
FYI we have a macro for hero now.

Don't change this though, let's just get it into production.

Reply:
Changed to the macro, though it outputs a `section` and I would prefer a `header` in this instance.